src/plugins/cosmos/components/UnbondingRow/UnbondingRow.tsx

```diff
@@ -0,0 +1,50 @@
+import { Flex } from '@chakra-ui/layout'
+import { Image } from '@chakra-ui/react'
+import pending from 'assets/pending.svg'
+import { Amount } from 'components/Amount/Amount'
+import { Text } from 'components/Text'
+import { BigNumber } from 'lib/bignumber/bignumber'
+
+type UnbondingRowProps = {
+  assetSymbol: string
+  cryptoUnbondedAmount: BigNumber
+  fiatRate: BigNumber
+  unbondingEnd: string // TODO: use timestamp and convert it to "x hours/days"
```

Comment:
any reason we can't do this now?

Reply:
Not really, IMO there just wasn't much value in this vs. a string until we wire up the data

Reply:
But nice one, makes the API more future proof at little cost - added!